goodtablesio/plugins/github/templates/github_home.html

```diff
@@ -0,0 +1,50 @@
+{% extends "layout.html" %}
+
+{% block contents %}
+<div class="container">
+
+  <h1>GitHub</h1>
+
+  <p>GitHub integration description</p>
+
+  <h2>Repos</h2>
+
+  {% if current_user.is_authenticated %}
+
+    {% if sync %}
+
+      <button disable class="btn btn-warning">Syncing account</button>
+      <script>
+        window.location.reload(5);
+      </script>
+
+    {% else %}
+
+      <div style="border-bottom: solid 1px #eee;">
+        <a href="/github/sync" class="btn btn-primary" style="width:120px;">Sync account</a>
+        <strong>Repositories</strong>
+      </div>
+
+      {% for repo in repos %}
+        <div style="border-bottom: solid 1px #eee;">
+          {% if repo.active %}
```

Comment:
It would be great if we could display them by organization and paginate them, but that can be done separately